src/EventStore.Client.Streams/IWriteResult.cs

```diff
@@ -1,6 +1,10 @@
-﻿namespace EventStore.Client {
+﻿using System;
+
+namespace EventStore.Client {
 	public interface IWriteResult {
+		[Obsolete("", true)]
```

Comment:
Perhaps add `"Please use NextExpectedStreamRevision instead. This property will be removed in a future version."`